README.md

```diff
@@ -205,6 +208,59 @@ All the attributions method presented below handle both **Classification** and *
   <i>methods with TF need a Tensorflow model. </i>
 </div>
 
+## 📞 Callable
+
+The model can be something else than a `tf.keras.Model` if it respects one of the following condition:
+- `model(inputs: np.ndarray)` return a `tf.Tensor` of shape $(N, L)$ where $N$ is the number of samples and $L$ the number of targets
+- The model has a `scikit-learn` API and has a `predict_proba` function
+- The model is a `xgboost.XGBModel` from the [XGBoost python library](https://xgboost.readthedocs.io/en/latest/python/python_intro.html)
+- The model is a [TF Lite model](https://www.tensorflow.org/api_docs/python/tf/lite). Note this feature is experimental.
+
+On the other hand, a PyTorch model can be used with method having Callable as type of model. In order to makes it work you should write a
+wrapper as follow:
+
+```python
```

Comment:
I don't know if we need to be that specific in the readme... Like this code block would better fit in a documentation page that we can link here no ? I would keep the list of 4 element and put a link on a doc page ?
What do you think ?

Reply:
Yes, I put into the docs!